mlrun/__main__.py

```diff
@@ -982,6 +982,13 @@ def logs(uid, project, offset, db, watch):
     "https://apscheduler.readthedocs.io/en/3.x/modules/triggers/cron.html#module-apscheduler.triggers.cron."
     "For using the pre-defined workflow's schedule, set --schedule 'true'",
 )
+# TODO: Remove in 1.6.0 --overwrite-schedule and keep --override-workflow
+@click.option(
+    "--overwrite-schedule",
```

Comment:
isnt it possible to add multiple strings (--x, --y, ..) to the same option vs add 2 seperate options?

Reply:
I didn't know about this, merged into one option